python/cuml/ensemble/randomforest.pyx

```diff
@@ -352,29 +354,37 @@ class RandomForestClassifier(Base):
     Parameters
     -----------
 
-    n_estimators : number of trees in the forest. default = 10
+    n_estimators : int (default = 10)
```

Comment:
Github won't allow me to comment on the earlier lines, but looking at the top of the docstring for this class, it says:
 "The model at the moment uses only numpy arrays as inputs." - that is not accurate.

Also, the text "The user is responsible for setting the various..." is not necessary, since that's true for basically every algorithm.